addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComTemperatureHumidityMessage.java

```diff
@@ -90,7 +109,7 @@ public byte toByte() {
 
     private final static List<RFXComValueSelector> supportedOutputValueSelectors = Arrays.asList();
 
-    public SubType subType = SubType.UNDEF;
+    public SubType subType = SubType.TH1;
```

Comment:
I agree, that's why I introduced UNKNOWNS, even they not in the rfxcom specs either.